source/viewmodel.ios.ts

```diff
@@ -365,7 +431,7 @@ class AssetPH extends Asset {
         return undefined;
     }
 
-    data(): Thenable<any> {
+    data(): Promise<any> {
```

Comment:
The Promise class implements the Thenable interface.
Usually it is better to stick with the Thenable interface.